drivers/hmis/app/graphql/types/hmis_schema/enums/project_config_type.rb

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Types
+  class HmisSchema::Enums::ProjectConfigType < Types::BaseEnum
+    graphql_name 'HmisProjectConfigType'
```

Comment:
```suggestion
    graphql_name 'ProjectConfigType'
```

No need to prefix with `Hmis` in the graphql schema